InstallSpinnaker.sh

```diff
@@ -311,6 +328,43 @@ function install_java() {
   fi
 }
 
+function install_platform_dependencies() {
+  local google_scopes=$(get_google_metadata_value "instance/service-accounts/default/scopes")
+
+  if [[ -z "$google_scopes" ]]; then
+     # Not on GCP
+     if [[ "x$GOOGLE_CLOUD_LOGGING" == "xtrue" ]] \
+        || [[ "x$GOOGLE_CLOUD_MONITORING" == "xtrue" ]]; then
+          if [[ ! -f /etc/google/auth/application_default_credentials.json ]];
+          then
+            echo "You may need to add Google Project Credentials."
+            echo "See https://developers.google.com/identity/protocols/application-default-credentials"
+          fi
+     fi
+  fi
+
+  if [[ "x$GOOGLE_CLOUD_LOGGING" == "xtrue" ]]; then
```

Comment:
I was following the practices elsewhere in the file. It's archaic and shouldnt be needed these days. in fact, it's probably only the -z test that [would have] needed it.